rgw/admin/radosgw.go

```diff
@@ -0,0 +1,157 @@
+package admin
```

Comment:
That doesn't really match with how the project is structured today. I just followed what seemed to be a "convention" from the other packages.

Reply:
Ok, t was mainly a suggestion, because I was confused when I was looking for the file that defines the `API` type. But if you think that's for good reasons, I will not object.